README.md

```diff
@@ -69,3 +69,53 @@ app.post('/contacts/:contactid', function (req, res) {
 	});
 });
 ```
+
+### Proposed item function reference
+Via the use of optional parameters and dedicated functions it is possible to retrieve anything from a single field on a single record to all fields on all records. Selection criteria and response timeouts may also be specified.
+
+The criteria parameter defines the query string to use to restrict the number of records that are returned. It followes the normal name=value&name=value... syntax
+
+The fields parameter defines which fields are to be output for each record. It expects a comma separated list of field names. Note that dataItemID, dataTypeID and lastUpdate will always be output.
```

Comment:
Passing in an array of field names might be clearer.  Then inside the library do fields.join() to add the commas.